tests/test_swift.py

```diff
@@ -87,35 +73,139 @@ def setUp(self):
         self.swift = swift.Swift(APP_INFOS)
 
     def test_init(self):
+        self.assertEqual(self.swift.appInfos, APP_INFOS)
         self.assertIsInstance(self.swift.mainWindow, QMainWindow)
         self.assertIsInstance(self.swift.centralWidget, QLabel)
         for name in APP_INFOS:
+            self.assertIn(name, self.swift._dockWidgets)
             self.assertIn(name, self.swift._apps)
         for channel in self.channels:
             self.assertIn(channel, self.swift._subscribers)
 
+    def test_app_names(self):
+        appNames = self.swift.appNames()
+        for name in APP_INFOS:
+            self.assertIn(name, appNames)
+
     def test_destroy_app(self):
         for name in APP_INFOS:
             self.swift.destroyApp(name)
             self.assertNotIn(name, self.swift._apps)
 
+    def test_update_frames(self):
+        self.swift._apps["app1"].frames.return_value = (QWidget(),)
+        self.swift.updateFrames("app1")
+
+    def test_channel_names(self):
+        channelNames = self.swift.channelNames()
+        for channelName in self.channels:
+            self.assertIn(channelName, channelNames)
+
+    def test_subscriber_names(self):
+        for channel in self.channels:
+            subscriberNames = self.swift.subscriberNames(channel)
+            for name, info in APP_INFOS.items():
+                if channel in info.channel:
+                    self.assertIn(name, subscriberNames)
+
```

Comment:
Missing test for `subscribe()`.

Reply:
As I wrote in PR, the test for `subscribe()` wasn't implemented because it is called when the `swift` instance is initialized and I checked if `self.swift._subscribers` is correct.

Do you think it is not enough?

Reply:
However, since the unit test should be independent, I think it should have its own unit test!
Maybe for the `createApp()` and `destroyApp()` as well.

Reply:
Okay! I will implement it.

Reply:
I have a question. 
The `createApp()` calls `subscribe()`, so I think if we test `createApp()`, it is not necessary to test `subscribe()`.
If you still want to sustain independently and separate into two tests, I think in the test for `createApp()`, `subscribe()` should be mocked.
What do you think of it?

Reply:
In `createApp()`, `subscribe()` and `addFrame()` are called, but I think it is enough to test all at once.
Do you agree with me..?

Reply:
I think mocking is the better option.
Reasons are:
1. In the future, we might change the implementation of `createApp()` or it could be removed.
2. Testing just `createApp()` might not cover enough cases for `subscribe()` (in general), since it could assume a special case, where `subscribe()` supports more general cases (just for example).

Reply:
Okay! Then do you think we need to test `_parseArgs()`, `_handleSwiftcall()`, and `_swiftcall()` individually?

Reply:
Oh, then I will do that after this PR!